### PR TITLE
[DC-3527] Updates to raw_rdr_export_qc checks

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 1]
```diff
@@ -798,55 +768,6 @@
...
- -- list all pids for whom no basics questions are found --
-SELECT *
-FROM `{{DATASET_ID}}.person`
-WHERE person_id not in (select person_id from pid_basics)
-""")
-query = tpl.render(DATASET_ID=new_rdr,
-                   BASICS_MODULE_CONCEPT_ID=BASICS_MODULE_CONCEPT_ID)
-execute(client, query)
-# -
-
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/raw_rdr_export_qc.py:849]: Removing basics queries from the raw notebook. These queries should be checked in the clean notebook instead and already exist there.

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 2]
```diff
@@ -949,27 +870,7 @@
...
-WHERE vocabulary_id = 'PPI' AND person_id NOT IN (
-SELECT DISTINCT person_id FROM `{{project_id}}.{{new_rdr}}.concept`
-JOIN `{{project_id}}.{{new_rdr}}.concept_ancestor` on (concept_id=ancestor_concept_id)
-JOIN `{{project_id}}.{{new_rdr}}.observation` on (descendant_concept_id=observation_concept_id)
-WHERE concept_class_id='Module'
-AND concept_name IN ('The Basics')
-AND questionnaire_response_id IS NOT NULL)
-''')
-query = tpl.render(new_rdr=new_rdr, project_id=project_id)
-execute(client, query)
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/raw_rdr_export_qc.py:970]: Removing basics queries from the raw notebook. These queries should be checked in the clean notebook instead and already exist there.

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 3]
```diff
@@ -949,27 +870,7 @@
...
-
-# ## Participants must be 18 years of age or older to consent
+# # Participants must be 18 years of age or older to consent
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/raw_rdr_export_qc.py:873]: Fix notebook formatting

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 4]
```diff
@@ -1033,89 +934,7 @@
...
-    that map to 1333342. Notify Odysseus that the issue still persists.
-    For pipeline, we can use cope_survey_semantic_version_map to diffrentiate COPE module versions,
-    so we can still move on. See DC-2641 for detail.
-'''
-
-render_message(df,
-               success_msg,
-               failure_msg,
-               failure_msg_args={'code_count': len(df)})
-# -
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/raw_rdr_export_qc.py:1116]: We are no longer expecting this change to the vocabulary.

File: data_steward/analytics/cdr_ops/raw_rdr_export_qc.py [hunk 5]
```diff
@@ -1250,39 +1069,66 @@
...
+FROM (SELECT DISTINCT observation_source_value, COUNT(*) as row_count
+    FROM `{{project_id}}.{{new_rdr}}.observation` o
+    WHERE observation_source_value = 'resultsconsent_wear'
+    GROUP BY 1) curr
+JOIN (SELECT DISTINCT observation_source_value, COUNT(*) row_count 
+    FROM `{{project_id}}.{{old_rdr}}.observation` o
+    WHERE observation_source_value = 'resultsconsent_wear'
+    GROUP BY 1) prev
+USING (observation_source_value)
+GROUP BY 1,2,3
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/raw_rdr_export_qc.py:1096]: Now that expectations for wear records are known. Updated the query to only show possible issues.

**Review by lledezma — APPROVED** [review]
Looks good!

**Pull request merged**